apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/block-inserter/index.js

```diff
@@ -0,0 +1,47 @@
+/* global fullSiteEditing */
+
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { render } from '@wordpress/element';
+import { Inserter } from '@wordpress/editor';
```

Comment:
`<Inserter />` has been moved to `@wordpres/block-editor` and it's triggering a deprecation warning.

```suggestion
import { Inserter } from '@wordpress/block-editor';
```

Reply:
https://github.com/Automattic/wp-calypso/pull/34425#discussion_r300818242